app/src/main/java/mozilla/lockbox/store/TelemetryStore.kt

```diff
@@ -61,6 +65,14 @@ open class TelemetryWrapper {
     open fun recordEvent(event: TelemetryEvent) {
         subject?.queueEvent(event)
     }
+
+    open fun scheduleUpload() {
+        val telem = subject ?: return
```

Comment:
nit, not related to this PR: I find the `subject` name a little confusing here, as we use it for the class under test in all the tests.

Reply:
I've no objections to changing it.  I think it was a pattern carried forward from somewhere.